ci/scripts/e2e-cassandra-sink-test.sh

```diff
@@ -36,27 +36,19 @@ risedev ci-start ci-sink-test
 # Wait cassandra server to start
 sleep 40
 
-echo "--- create cassandra table"
-curl https://downloads.apache.org/cassandra/4.1.3/apache-cassandra-4.1.3-bin.tar.gz  --output apache-cassandra-4.1.3-bin.tar.gz
-tar xfvz apache-cassandra-4.1.3-bin.tar.gz
+echo "--- install cassandra"
+curl https://downloads.apache.org/cassandra/4.1.5/apache-cassandra-4.1.5-bin.tar.gz  --output apache-cassandra-4.1.5-bin.tar.gz
```

Comment:
To close #17855, we should pull the latest version of cassandra sink instead. Otherwise each time it bumps, our ci fails because we can't pull the cassnadra binary.

You can see `get_latest_kafka_version` and `get_latest_kafka_download_url` in `ci/scripts/common.sh`